src/wallet/utils.rs

```diff
@@ -30,16 +28,15 @@ pub(crate) const BLOCKS_TIMELOCK_THRESHOLD: u32 = 500000000;
 
 /// Trait to check if a value is below the dust limit
 // we implement this trait to make sure we don't mess up the comparison with off-by-one like a <
-// instead of a <= etc. The constant value for the dust limit is not public on purpose, to
-// encourage the usage of this trait.
+// instead of a <= etc.
```

Comment:
Maybe we can add a doc line here saying "We are perform dust value calculation for a given scriptpubkey using rust-bitcoin, to keep it compatible with network dust fee rate". Not needed but can be helpful for new readers.

Reply:
You're right that would be nice for new readers--I'll include a similar message. Thanks :)